src/main/groovy/org/labkey/gradle/util/BuildUtils.groovy

```diff
@@ -788,6 +790,8 @@ class BuildUtils
 
     static String getModuleProjectPath(ModuleDependency dependency)
     {
+        if (LIMS_MODULES.contains(dependency.getName()))
+            return ":server:modules:limsModules:" + dependency.getName()
```

Comment:
Are there any similar references to `premiumModules` that we should hack too?

Reply:
I found no other references.